demos/src/simple-header.mustache

```diff
@@ -1,157 +1,144 @@
-                    {{#hasMenu}}
-                        <a href="#o-header-drawer" class="o-header__top-link o-header__top-link--menu" aria-controls="o-header-drawer">
-                            <span class="o-header__top-link-label">Menu</span>
-                        </a>
-                    {{/hasMenu}}
-                    <a href="#o-header-search" class="o-header__top-link o-header__top-link--search" aria-controls="o-header-search-js">
-                        <span class="o-header__top-link-label">Search</span>
-                    </a>
-                </div>
-                <div class="o-header__top-column o-header__top-column--center">
-                    <a class="o-header__top-logo" href="/" title="Go to Financial Times homepage">
-                        <span class="o-header__visually-hidden">Financial Times</span>
-                    </a>
-                    {{#nav}}
-                        <nav id="o-header-nav-desktop" class="o-header__nav o-header__nav--desktop" role="navigation">
-                                <ul class="o-header__nav-list o-header__nav-list--left">
-                                    {{#desktop}}
-                                        <li class="o-header__nav-item">
-                                            <a class="o-header__nav-link o-header__nav-link--primary" href="{{href}}" aria-selected="{{selected}}" id="o-header-link-{{index}}" {{#hasMega}}aria-controls="o-header-mega-{{index}}"{{/hasMega}}>{{name}}</a>
-                                        </li>
-                                    {{/desktop}}
-                                </ul>
-                        </nav>
-                    {{/nav}}
+	<div class="o-header__row o-header__top">
+		<div class="o-header__container">
+			<div class="o-header__top-wrapper">
+
+				<div class="o-header__top-column o-header__top-column--left">
+					{{#top.hasMenu}}
+					<a href="#o-header-drawer" class="o-header__top-link o-header__top-link--menu" aria-controls="o-header-drawer">
+						<span class="o-header__top-link-label">Menu</span>
+					</a>
+					{{/top.hasMenu}}
+					<a href="#o-header-search" class="o-header__top-link o-header__top-link--search" aria-controls="o-header-search-sticky">
+						<span class="o-header__top-link-label">Search</span>
+					</a>
+				</div>
+
+				<div class="o-header__top-column o-header__top-column--center">
+					<a class="o-header__top-logo" href="/" title="Go to Financial Times homepage">
+						<span class="o-header__visually-hidden">Financial Times</span>
+					</a>
+				</div>
+
+				<div class="o-header__top-column o-header__top-column--right">
+					{{#top.hasMyFT}}
+					<a class="o-header__top-link o-header__top-link--myft" href="/myft">
+						<span class="o-header__visually-hidden">myFT</span>
+					</a>
+					{{/top.hasMyFT}}
+				</div>
+			</div>
+
+		</div>
+	</div>
+
+	<div id="o-header-search-sticky" class="o-header__row o-header__search o--if-js" role="search" data-o-header-search>
```

Comment:
On Next at least we'll be running both the full header _and_ sticky header so this avoids any naming collisions